pubsub/redis.go

```diff
@@ -38,13 +42,79 @@ func NewRedisSubscriber(node *node.Node, url string, channel string) RedisSubscr
 // Start connects to Redis and subscribes to the pubsub channel
 func (s *RedisSubscriber) Start() error {
 	// Check that URL is correct first
-	_, err := url.Parse(s.url)
+	redisUrl, err := url.Parse(s.url)
 
 	if err != nil {
 		return err
 	}
 
+	var sntnl *sentinel.Sentinel
```

Comment:
What is the reason for that explicit declaration instead of good ol' shorthand syntax?

Reply:
that reason is gone now. thanks. updated it